charts/fybrik/templates/openmetadata-connector/openmetadata-connector-configmap.yaml

```diff
@@ -0,0 +1,24 @@
+{{- $autoFlag := and .Values.coordinator.enabled (eq .Values.coordinator.catalog "openmetadata") }}
+{{- if include "fybrik.isEnabled" (tuple .Values.openmetadataConnector.enabled $autoFlag) }}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fybrik.fullname" . }}-openmetadata-connector-configmap
+data:
+  conf.yaml: |- 
+{{- if .Values.config_override }}
+{{ .Values.config_override  | indent 4}}
+{{- else }}
+    openmetadata_endpoint: {{ .Values.openmetadataConnector.openmetadata_endpoint | default "openmetadata.fybrik-system:8585/api" }}
```

Comment:
I think that all defaults should be in values.yaml and not here

Reply:
done.